[0.006s] Invoking command in '/home/bradenmeyers/ros2ws/src/holoocean_interfaces/build/holoocean_interfaces': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble:/home/bradenmeyers/ros2ws/install/py_pubsub:/home/bradenmeyers/ros2ws/install/python_parameters:/home/bradenmeyers/ros2ws/install/my_package /usr/bin/cmake /home/bradenmeyers/ros2ws/src/holoocean_interfaces -DCMAKE_INSTALL_PREFIX=/home/bradenmeyers/ros2ws/src/holoocean_interfaces/install/holoocean_interfaces
[0.104s] -- The C compiler identification is GNU 11.4.0
[0.195s] -- The CXX compiler identification is GNU 11.4.0
[0.206s] -- Detecting C compiler ABI info
[0.280s] -- Detecting C compiler ABI info - done
[0.285s] -- Check for working C compiler: /usr/bin/cc - skipped
[0.286s] -- Detecting C compile features
[0.287s] -- Detecting C compile features - done
[0.291s] -- Detecting CXX compiler ABI info
[0.373s] -- Detecting CXX compiler ABI info - done
[0.378s] -- Check for working CXX compiler: /usr/bin/c++ - skipped
[0.378s] -- Detecting CXX compile features
[0.378s] -- Detecting CXX compile features - done
[0.383s] -- Found ament_cmake: 1.3.9 (/opt/ros/humble/share/ament_cmake/cmake)
[0.519s] -- Found Python3: /usr/bin/python3 (found version "3.10.12") found components: Interpreter 
[0.588s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[0.591s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.597s] -- Found rosidl_adapter: 3.1.5 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.634s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.662s] -- Found std_msgs: 4.2.4 (/opt/ros/humble/share/std_msgs/cmake)
[0.707s] -- Found ament_lint_auto: 0.12.11 (/opt/ros/humble/share/ament_lint_auto/cmake)
[0.895s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[0.895s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.895s]   holoocean_interfaces --arguments-file
[0.895s]   /home/bradenmeyers/ros2ws/src/holoocean_interfaces/build/holoocean_interfaces/rosidl_adapter__arguments__holoocean_interfaces.json
[0.895s]   --output-dir
[0.895s]   /home/bradenmeyers/ros2ws/src/holoocean_interfaces/build/holoocean_interfaces/rosidl_adapter/holoocean_interfaces
[0.895s]   --output-file
[0.895s]   /home/bradenmeyers/ros2ws/src/holoocean_interfaces/build/holoocean_interfaces/rosidl_adapter/holoocean_interfaces.idls)
[0.895s]   returned error code 1:
[0.895s] 
[0.895s]   Error processing 'float32[][] image' of
[0.895s]   'holoocean_interfaces/ImagingSonar': ''float32[]' is an invalid message
[0.895s]   name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$''
[0.895s] 
[0.895s]   Traceback (most recent call last):
[0.895s] 
[0.895s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[0.895s]       return _run_code(code, main_globals, None,
[0.895s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[0.895s]       exec(code, run_globals)
[0.895s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.895s]       sys.exit(main())
[0.895s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[0.895s]       abs_idl_file = convert_to_idl(
[0.895s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
[0.895s]       return convert_msg_to_idl(
[0.895s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
[0.895s]       msg = parse_message_string(package_name, input_file.stem, content)
[0.895s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[0.896s]       Type(type_string, context_package_name=pkg_name),
[0.896s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 277, in __init__
[0.896s]       super(Type, self).__init__(
[0.896s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[0.896s]       raise InvalidResourceName(
[0.896s] 
[0.896s]   rosidl_adapter.parser.InvalidResourceName: 'float32[]' is an invalid
[0.896s]   message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.896s] 
[0.896s] Call Stack (most recent call first):
[0.896s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.896s]   CMakeLists.txt:29 (rosidl_generate_interfaces)
[0.896s] 
[0.896s] [0m
[0.896s] -- Configuring incomplete, errors occurred!
[0.896s] See also "/home/bradenmeyers/ros2ws/src/holoocean_interfaces/build/holoocean_interfaces/CMakeFiles/CMakeOutput.log".
[0.899s] Invoked command in '/home/bradenmeyers/ros2ws/src/holoocean_interfaces/build/holoocean_interfaces' returned '1': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/opt/ros/humble:/home/bradenmeyers/ros2ws/install/py_pubsub:/home/bradenmeyers/ros2ws/install/python_parameters:/home/bradenmeyers/ros2ws/install/my_package /usr/bin/cmake /home/bradenmeyers/ros2ws/src/holoocean_interfaces -DCMAKE_INSTALL_PREFIX=/home/bradenmeyers/ros2ws/src/holoocean_interfaces/install/holoocean_interfaces
